ietf/templates/doc/search/status_columns.html

```diff
@@ -77,7 +77,7 @@
             <br>
             Action Holder{{ doc.documentactionholder_set.all|pluralize }}:
             {% for action_holder in doc.documentactionholder_set.all %}
-                {% person_link action_holder.person title=action_holder.role_for_doc %}{% if action_holder|action_holder_badge %} {{ action_holder|action_holder_badge }}{% endif %}{% if not forloop.last %},{% endif %}
+                {% person_link action_holder.person title=action_holder.role_for_doc %}{% if forloop.last %}{% if action_holder|action_holder_badge %} {{ action_holder|action_holder_badge }}{% endif %}{% else %},{% endif %}
```

Comment:
This doesn't work (as the test failure at ietf.group.tests_info.GroupPagesTests.test_group_documents points out).

It is possible that not all the action holders have the same start time.

In this test, the first action holder was the one that had been added a long time ago, the second had been just added, so no badge got displayed.

Reply:
(To be fair, my suggestion on the associated ticket implied one badge here might work, but on review of the code, it doesn't)